### ISSUE TITLE
`Communication`: Add an overview over all your threads

### ISSUE THREAD

**Issue by PaRangger** [ls1intum/Artemis#9951]
### Is your feature request related to a problem?

Currently it is easy to loose the overview over the threads I am currently writing in and whether i got replies or not.

### Describe the solution you'd like

Add a possibility to go through all threads that you have created messages in. E.g. the "Threads" tab in slack.

**Comment by LuisL02** [comment]
This has not been implemented yet

**Comment by Jess-hub09** [comment]
I agree, this is a very relevant issue, and I actually wanted to open a feature request for the same reason. It’s extremely easy to lose track of replies in threads, especially since there is no notification or visual indicator that a new message in a thread is unread. Only direct mentions (tags) in the treads show up reliably. Having a “My Threads” or “Participated Threads” view, similar to Slack or getting a notification if a student comments in the same thread where I commented, would really help keep an overview.
